src/frontend/parser.mly

```diff
@@ -543,7 +543,10 @@ common_expression:
          && ( String.is_suffix ~suffix:"_lpdf" id.name
               || String.is_suffix ~suffix:"_lupdf" id.name
               || String.is_suffix ~suffix:"_lpmf" id.name
-              || String.is_suffix ~suffix:"_lupmf" id.name )
+              || String.is_suffix ~suffix:"_lupmf" id.name
+              || String.is_suffix ~suffix:"_cdf" id.name
+              || String.is_suffix ~suffix:"_lcdf" id.name
+              || String.is_suffix ~suffix:"_lccdf" id.name )
```

Comment:
This could be replaced by

```ocaml
List.length args = 1 &&
List.exists ~f:(fun x -> String.is_suffix ~suffix:x id.name) 
Utils.conditioning_suffice
```